homeassistant/components/yeelight/__init__.py

```diff
@@ -163,10 +160,10 @@ async def async_setup(hass: HomeAssistant, config: dict) -> bool:
     }
 
     # Import manually configured devices
-    for ipaddr, device_config in config.get(DOMAIN, {}).get(CONF_DEVICES, {}).items():
-        _LOGGER.debug("Importing configured %s", ipaddr)
+    for ip_addr, device_config in config.get(DOMAIN, {}).get(CONF_DEVICES, {}).items():
```

Comment:
Should it be `host` instead of `ip_addr`, assuming that hostnames are also accepted?